extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/FSInternetRadioDiscoveryParticipant.java

```diff
@@ -91,17 +102,31 @@
         // as reported in: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/7
         final Set<String> revoRadios = new HashSet<String>();
         SUPPORTED_RADIO_MODELS.put("REVO TECHNOLOGIES LTD", revoRadios);
-        revoRadios.add("SUPERCONNECT");
```

Comment:
Did this id change? Do you have a reference?

Reply:
The discovery maps contain **manufacturer** or **model name** as key and the **model numbers** as values.
I fixed some entries that mapped **manufacturer** to **model name** (the model number was entirely missing), e.g. 'S10' instead of 'SUPERCONNECT'.